pyproject.toml

```diff
@@ -99,11 +71,59 @@ pep621 = "commitizen.providers:Pep621Provider"
 poetry = "commitizen.providers:PoetryProvider"
 scm = "commitizen.providers:ScmProvider"
 
-[tool.poetry.plugins."commitizen.scheme"]
+[project.entry-points."commitizen.scheme"]
 pep440 = "commitizen.version_schemes:Pep440"
 semver = "commitizen.version_schemes:SemVer"
 semver2 = "commitizen.version_schemes:SemVer2"
 
+[build-system]
+requires = ["poetry-core>=2.0"]
+build-backend = "poetry.core.masonry.api"
+
+
+[tool.commitizen]
```

Comment:
maybe we can use the `version_scheme = "pep440"` now?

Reply:
right! I just figure out I'm still using old old commitizen feature in my project template 🤦‍♂️ will update it!